File: app/src/main/java/com/example/exemplocardview/componenteCardView/CardQuestionInterface.java
package com.example.exemplocardview.componenteCardView;

interface CardQuestionInterface {

    void openCard();

    void closeCard();

    boolean isCardOpen();

    boolean isCardClose();

    void setTextHeader(String text_title_header);

    String getTextHeader();

    void setTextQuestion(String text_question);

    String getTextQuestion();

    void setTextRemarks(String text_remarks);

    String getTextRemarks();

    void setLikeIconActivated();

    void setLikeIconDeactivated();

    void setUnlikeIconActivated();

    void setUnlikeIconDeactivated();

    boolean isLikeSet();

    boolean isUnlikeSet();

    void resetComponent();
}

File: app/src/main/java/com/example/exemplocardview/componenteCardView/CardQuestionInterface.kt
package com.example.exemplocardview.componenteCardView

internal interface CardQuestionInterface {
    fun openCard()
    fun closeCard()
    fun isCardOpen(): Boolean
    fun isCardClose(): Boolean
    fun textHeader(text : String?)
    fun textQuestion(text: String?)
    fun textRemarks(text: String?)
    fun setLikeIconActivated()
    fun setLikeIconDeactivated()
    fun setUnlikeIconActivated()
    fun setUnlikeIconDeactivated()
    fun isLikeSet(): Boolean
    fun isUnlikeSet(): Boolean
    fun resetComponent()
}